pyperf/_collect_metadata.py

```diff
@@ -11,8 +11,12 @@
     resource = None
 
 try:
+    from pyperf._utils import IS_FREE_THREADING
     # Optional dependency
-    import psutil
+    if IS_FREE_THREADING:
```

Comment:
Please add a link to the issue to explain the skip. Same remark for similar changes.